apps/api/src/mail/mail.service.ts

```diff
@@ -77,7 +77,7 @@ export class MailService {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   public async sendMailable(mailable: Mailable<any>): Promise<ChesResponse> {
     const mailOptions: Partial<MailOptions> = {
-      from: process.env.MAIL_FROM,
+      from: process.env.MAIL_FROM || 'EHPRDoNotReply@gov.bc.ca',
```

Comment:
Why do we need this change?

Reply:
I'll remove it. I didn't have the environment variable in my local, so CHES was complaining. Also, it doesn't hurt to have a backup just in case the variable isn't defined in an environment